Documentation/Walkthrough.md

```diff
@@ -0,0 +1,206 @@
+# Getting started
+
+The definition and implementation of data and service contracts is the same as WCF. The major difference is in the definition of the host which is now based on ASP.NET Core, and the ceremony of how a service is exposed. The following is based on .NET 6, but the same steps apply to other versions of .NET.
+
+## Defining the service
+
+### 1. Create an ASP.NET Core Empty application, this provides the host for the service.
+
+*Visual Studio:*
+ ![Project Template](images/empty-web-project.png)
+
+*Command Line:*
+
+```cli
+mkdir CoreWCFDemoServer
+dotnet new web -n CoreWCFDemoServer -o CoreWCFDemoServer
+```
+
+### 1. Add references to the CoreWCF Nuget Packages
+
+*Visual Studio:*
+
+Using the package Manager console, add: 
+
+- CoreWCF.Primitives
+- CoreWCF.Http
+
+![Package Manager Console](images/nuget.png)
+
+*Command Line:*
+
+Edit the project file and add:
+
+```xml
+<ItemGroup>
+  <PackageReference Include="CoreWCF.Primitives" Version="1.0.0" />
+  <PackageReference Include="CoreWCF.Http" Version="1.0.0" />
+</ItemGroup>
+```
+### 1. Create the Service Contract and Data Contract definitions
+
+These are defined the same as with WCF. When modernizing projects, this code can remain largely unchanged.
```

Comment:
Would it be worth mentioning that if you need to reuse your contracts for server and client in a shared assembly, that CoreWCF supports using the System.ServiceModel versions of the attributes that can be found in the System.ServiceModel.Primitives client package? Or is that over complicating things?

Reply:
It's in the video you linked to at the end of your blog and is in a blog post.